pom.xml

```diff
@@ -89,6 +89,7 @@
   </distributionManagement>
 
   <properties>
+    <argLine />
```

Comment:
Why this?

Reply:
some left over from some testing I made.
removed